scripts/gen/windows-resources

```diff
@@ -6,6 +6,7 @@
 set -eu -o pipefail
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=/go/src/github.com/docker/cli/scripts/build/.variables
```

Comment:
I believe this can be removed, as well as the other 4 instances of it in the `script/build` files